code/modules/projectiles/projectile/bullets.dm

```diff
@@ -184,10 +184,21 @@
 	penetrating = 1
 
 /obj/item/projectile/bullet/rifle/a762
-	damage = 25
+	damage = 30
 
 /obj/item/projectile/bullet/rifle/a556
-	damage = 30
+	damage = 25
+
+/obj/item/projectile/bullet/rifle/mk43
+	damage = 20
+
+/obj/item/projectile/bullet/rifle/mk43/rubber
+	name = "rubber bullet"
+	check_armour = "melee"
+	damage = 6
+	agony = 50

```

Comment:
I'd make this a bit higher. Fifty agony is like, not that much. Just ten above a .45 rubber bullet.